src/core/core.animations.js

```diff
@@ -67,54 +71,67 @@ export default class Animations {
 	 */
   _createAnimations(target, values) {
     const animatedProps = this._properties;
+    const pathAnimatedProps = this._pathProperties;
     const animations = [];
     const running = target.$animations || (target.$animations = {});
     const props = Object.keys(values);
     const date = Date.now();
-    let i;
 
-    for (i = props.length - 1; i >= 0; --i) {
-      const prop = props[i];
-      if (prop.charAt(0) === '$') {
-        continue;
-      }
-
-      if (prop === 'options') {
-        animations.push(...this._animateOptions(target, values));
-        continue;
-      }
-      const value = values[prop];
+    const manageItem = function(tgt, vals, prop, subProp) {
+      const key = subProp || prop;
+      const value = vals[key];
       let animation = running[prop];
       const cfg = animatedProps.get(prop);
 
       if (animation) {
         if (cfg && animation.active()) {
           // There is an existing active animation, let's update that
           animation.update(cfg, value, date);
-          continue;
-        } else {
-          animation.cancel();
+          return;
         }
+        animation.cancel();
       }
       if (!cfg || !cfg.duration) {
         // not animated, set directly to new value
-        target[prop] = value;
-        continue;
+        tgt[key] = value;
+        return;
       }
-
-      running[prop] = animation = new Animation(cfg, target, prop, value);
+      running[prop] = animation = new Animation(cfg, tgt, key, value);
       animations.push(animation);
+    };
+
+    let i;
+    for (i = props.length - 1; i >= 0; --i) {
+      const prop = props[i];
+      if (prop.charAt(0) === '$') {
+        continue;
+      }
+      if (prop === 'options') {
+        animations.push(...this._animateOptions(target, values));
+        continue;
+      }
+      if (pathAnimatedProps.has(prop)) {
+        pathAnimatedProps.forEach(function(item) {
+          const newTarget = getInnerObject(target, item);
+          const newValues = getInnerObject(values, item);
+          if (newTarget && newValues) {
+            manageItem(newTarget, newValues, item.prop, item.key);
+          }
+        });
+      } else {
+        manageItem(target, values, prop);
+      }
```

Comment:
Looks like each "path" prop would be resolved multiple times if there are multiple "path" props?

I think most of the new helpers could be omitted(?):

```suggestion
      if (prop.includes('.')) {
        const newTarget = resolveObjectKey(target, prop);
        const newValues = newTarget && resolveObjectKey(values, prop);
        if (newValues) {
          manageItem(newTarget, newValues, prop, prop.split('.').pop());
        }
      } else {
        manageItem(target, values, prop);
      }
```

Did I miss something?

Reply:
Furthermore, I cannot use `resolveObjectKey` helper because this function is returning the value of the key and not the object where the key belongs to and therefore I cannot reuse the `manageItem` function for both normal and path anim options, where the function needs the object with key.

Reply:
@kurkle I have improved a bit removing the for Each statemenet. Anyway forEach should be needed. Need to time to review. Apologize

Reply:
@kurkle now I recall why we need to scan the map but was wrong. Now I fixed.

The map has:

K = string = the first item of the path (i.e. `'font'` if animation properties config has `'font.size'`)
V = array = the array of objects with the animation properties config

The V is an array because you can configure more than 1 anim prop as path (i.e. `['font.size', 'font.weight']`)